pages/dr9/issues.rst

```diff
@@ -57,3 +57,24 @@ A list of the affected bricks `is available here`_.
 `Blobmodel images`_ were intended to be compressed, and are named with a ``.fits.fz`` suffix, but they are not actually compressed.
 
 .. _`Blobmodel images`: ../files/#image-stacks-region-coadd
+
+
+``NOBS`` differs between the Tractor catalogs and random catalogs
+-----------------------------------------------------------------
+
+Quantities named ``nobs_x`` (where ``x`` is any of the Legacy Surveys imaging bands) in the `Tractor catalogs`_ (and derived
+products such as the sweep files) are slightly different to the ``NOBS_X`` quantities in the `random catalogs`_.
```

Comment:
Some people prefer "different from", other people prefer "different than", but I don't think "different to" is correct.

Reply:
The expression "different to" is British English. I guess I haven't quite scrubbed all of my grammar from across-the-pond. 

I'm happy to change it, though, and have!